src/main/java/Events.java

```diff
@@ -0,0 +1,19 @@
+public class Events extends Task{
+    protected String from;
+    protected String to;
+    public Events(String description, String from, String to){
+        super(description);
+        identity = "[E]";
+        this.from = from;
+        this.to = to;
+    }
+    public String getFrom() {
```

Comment:
Consider adding comments to specify the use of the functions

Reply:
will do at the end of project